packages/kernel-5.10/kernel-5.10.spec

```diff
@@ -237,6 +279,18 @@ rm -f %{buildroot}%{kernel_libdir}/build %{buildroot}%{kernel_libdir}/source
 ln -sf %{_usrsrc}/kernels/%{version} %{buildroot}%{kernel_libdir}/build
 ln -sf %{_usrsrc}/kernels/%{version} %{buildroot}%{kernel_libdir}/source
 
+# Install a copy of System.map so that module dependencies can be regenerated.
+install -p -m 0600 System.map %{buildroot}%{kernel_libdir}
```

Comment:
Why is this needed for 5.10 and 5.15, and not 6.1? I don't see anything depmod-specific across our kernels, but may be missing something.

Reply:
Disregard, I see we already have this installed for 6.1 :)